src/main/java/hu/oe/nik/szfmv/automatedcar/model/Position.java

```diff
@@ -1,40 +1,70 @@
 package hu.oe.nik.szfmv.automatedcar.model;
 
+import hu.oe.nik.szfmv.automatedcar.xml.converter.IntegerConverter;
+
+import javax.xml.bind.annotation.*;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "Position")
 public class Position {
+
+    @XmlAttribute(name = "x")
+    @XmlJavaTypeAdapter(value = IntegerConverter.class, type = int.class)
```

Comment:
lehetne inkább Integer és akkor nem kell a type = int (mint a transformban)

Reply:
@pintergreg javasolta, hogy ha megoldható maradjunk az int és double-nél. Szerintem jó meglátás volt, mert elég sok mindent kellett volna refaktorálni, minden csapatot érintett volna. Funkcionalitás szempontjából pedig mindegy lesz, hogy osztály-e a változó, mert null-t amúgy se állítunk be sehol. Ha megnézed a konvertereket, azok is 0-át adnak vissza invalid input vagy hiba esetén, hogy ne szálljon el a többiek kódja.

Reply:
Rendben, de ezzel eldobunk sok java specifikus előnyt